iquip/apps/scheduler.py

```diff
@@ -30,6 +34,23 @@ def _dismiss_items(layout: Optional[QLayout] = None):
                 _dismiss_items(item.layout())
 
 
+def _run_thread_with_worker(worker: QObject, parent: Optional[QObject] = None):
+    """Runs another thread with given worker.
```

Comment:
The name seems awkward since it does not actually run the thread, but create and prepare the threaded worker.